### ISSUE TITLE
Specific conv bias init value (-2.19) for the keypoints in author's pytorch code. Line 132 in your code.

### ISSUE THREAD

**Issue by unlabeledData** [Stick-To/CenterNet-tensorflow#24]
The author use a special init value of the last keypoints conv's bias. 
And he or she said it is because of the focal loss.
So do you use this setting?
I think the same layer should be in your code is Line 132 in CenterNet.py.
Line132: keypoints = self._conv_bn_activation(features, self.num_classes, 3, 1, None)

**Comment by Stick-To** [comment]
I have not read the official code, I implement it recording to paper

**Comment by unlabeledData** [comment]
OK. Thank you.
Could you share your evaluation results on VOC? 
It's much kinder if you can share your training loss logs.

**Comment by Stick-To** [comment]
sorry, I did not keep them
